software-update/fpga_list.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+ls /etc/fpga/
+mount /dev/sda1 /mnt
+mkdir /temp_fpga
+cp /mnt/*.jam /temp_fpga
+umount /mnt
```

Comment:
We are unable to identify any cockpit APIs that help with the uploading mechanism ie transferring of the file to the compute board via a 'browse the file explorer and upload' option. Kindly suggest.